### PR TITLE
fix possible RCE in jsonpath-plus

### PR DESCRIPTION
**Pull request by rluvaton**
https://github.com/JSONPath-Plus/JSONPath/issues/182

### CODE REVIEW

File: src/json_path_test.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai';
+import { jsonpath } from './json_path';
+
+describe('jsonpath', () => {
```
Comment (brendandburns) [inline src/json_path_test.ts:4]: Can you add a test like this to the gcp_auth test also?

Thanks

**Comment by rluvaton** [comment]
to be honest after I found that issue I looked at the top most packages that use that and fixed that in them to have better security in our ecosystem

The RCE was meant to be in the package itself (even though it's a feature)...

File: src/json_path_test.ts (hunk 1 above)
Comment (rluvaton) [reply src/json_path_test.ts:4]: done 😄

**Pull request merged**